README.md

```diff
@@ -59,11 +65,10 @@ The answer is a JSON dict section -> proxy. The dict is empty if the file couldn
 $ echo $JSON_PROXY
 {"xcp-ng-base": "http://192.168.100.82:3142", "xcp-ng-updates": "_none_", "xcp-ng-extras": "_none_", "xcp-ng-extras_testing": "_none_", "xcp-ng-updates_testing": "http://192.168.100.82:3142"}
 $ xe host-call-plugin host-uuid=${HOST_UNDER_TEST_UUID} plugin=updater.py fn=set_proxies args:proxies="'$JSON_PROXY'"
-{"status": true}
 ```
 The `proxies` parameter is a JSON dict section-> proxy for the list of sections whose proxy we want to alter.
 
-The returned value should be the JSON value `{"status": true}` or `{"status": false, "error": "<message>"}`.
+The returned value should be the JSON value `''` in case of success.
```

Comment:
I'm not entirely convinced that returning `''` rather than an empty string is better. XO is not necessarily supposed to be the only consumer of that kind of plugins. While I understand that JSON is useful for formatted output, I think I'd prefer we'd stick to the upstream way for functions that return nothing.

Reply:
`JSON.dumps('') = ''`

Reply:
As discussed, not true :)